usagov/usagov-2021
# USAGOV-934: USAGOV-934 - Remove specific node version from CircleCI

https://cm-jira.usa.gov/browse/USAGOV-934

## Description
Remove the step that forces the installation of the specific node-version for the linter.  

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions
Switch to this branch. To be through, I suppose you could delete all docker processes found with `docker ps -a`, and then run `bin/npm run lint`. We just need to confirm the linter still runs and does not crash due to a lack of Node and/or having the wrong version of Node installed. As long as the Linter runs without crashing, we are good I believe.

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
